tests/hpc/ww4_await_pxe_install.pm

```diff
@@ -17,17 +17,13 @@ use mm_tests;
 use POSIX 'strftime';
 
 sub run ($self) {
-    if (check_screen('pxe-start')) {
-        send_key "ctrl-B";
-        save_screenshot();
-        record_info "pxe", "wait for controller";
-    }
+    assert_screen('qemu-no-bootable-device', timeout => 90);
     mutex_wait "ww4_ready";
     barrier_wait('WWCTL_READY');
```

Comment:
Waiting for mutex and creating barrier  just after is not correct as well, will be handled in another future PR.